src/main/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTask.java

```diff
@@ -417,30 +418,24 @@ private RootModule createRootModule() {
      */
     private Properties createOverridingProperties() {
         final Properties returnValue = new Properties();
-
         // Load the properties file if specified
         if (properties != null) {
-            try (InputStream inStream = Files.newInputStream(properties.toPath())) {
-                returnValue.load(inStream);
+            try (InputStream input = Files.newInputStream(properties.toPath())) {
+                returnValue.load(input);
             }
             catch (final IOException ex) {
                 throw new BuildException("Error loading Properties file '"
                         + properties + "'", ex, getLocation());
             }
         }
-
         // override with Ant properties like ${basedir}
-        final Map<String, Object> antProps = getProject().getProperties();
-        for (Map.Entry<String, Object> entry : antProps.entrySet()) {
```

Comment:
i need to stop doing this as its kind of the normal pattern here and all around. Considering creating a [rewrite recipe](https://docs.openrewrite.org/recipes) out of this to be fixable for others as well.